CustomerData/DefaultItem.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Alma\MonthlyPayments\CustomerData;
+
+use Alma\MonthlyPayments\Helpers\InsuranceHelper;
+use Alma\MonthlyPayments\Helpers\Logger;
+use Magento\Framework\App\ObjectManager;
+
+class DefaultItem extends \Magento\Checkout\CustomerData\DefaultItem
+{
+    protected function doGetItemData(): array
```

Comment:
Can't name getItemData because the class extend native DefaultItem class?

Reply:
It's only the namespace that change, not the class name i'm extending